daml-lf/interpreter/src/main/scala/com/digitalasset/daml/lf/speedy/SValue.scala

```diff
@@ -234,23 +234,25 @@ object SValue {
     override def toString: String = s"SBigNumeric($value)"
   }
   object SBigNumeric {
-    // TODO https://github.com/digital-asset/daml/issues/8719
-    //   Decide what are the actual bound for BigDecimal
-    val MaxScale = 1 << 10
-    val MaxPrecision = MaxScale << 2
+    val MaxPrecision = 1 << 16
+    val MaxScale = MaxPrecision / 2
+    val MinScale = -MaxPrecision / 2 + 1
```

Comment:
Can we shift the `scale` range down by 1, so it's:

```
    val MaxScale = MaxPrecision / 2 - 1
    val MinScale = -MaxPrecision / 2
```

Instead? That way the `scale` range can be described exactly as the range of a signed 16-bit integer (`i16` in rust).  It's a range that most programmers are more familiar with.



<img width="1233" alt="image" src="https://user-images.githubusercontent.com/63245722/113565913-b0c99000-9603-11eb-9985-5828754d1670.png">

Reply:
We agree offline to keep 
```
    val MaxScale = MaxPrecision / 2
    val MinScale = -MaxPrecision / 2 + 1
```
as this preserves the nice symmetry: